constraints/src/main/java/gov/nasa/jpl/aerie/constraints/tree/ListExpressionAt.java

```diff
@@ -22,9 +22,6 @@ public DiscreteProfile evaluate(final SimulationResults results, final Interval
     //Even if the expression returns a discrete profile, this profile has a constant value, a list of serialized values,
     //at the singleton timepoint it has been computed at.
     //The instantiation and use of valueAt is necessary here because the profile are not only discrete but also "continuous" in the form of linear profiles.
-    if(!bounds.isSingleton()){
-      throw new Error("The ListExpressionAt node should be used only with singleton bounds.");
-    }
```

Comment:
Why has  the `isSingleton` check been removed from here and `StructExpressionAt`?

Reply:
Never mind, it's because of the relaxation 🤦‍♀️. 

Couple of follow-ups:
1) The comments directly above the removals are now out of date, as they refer to this singleton enforcement.
2) Follow up question: is `Interval.FOREVER` the only non-singleton value permitted?

Reply:
2. No it is not the only possible value at this level. It's only in the instantiate method in `SchedulingActivityDirective` that it does not make sense to restrict the evaluation interval.